### PR TITLE
Implement makeKey static method

### PR DESCRIPTION
**Pull request by narthur157**
I ended up implementing this in a project I'm doing, so I figured I'd make a PR for it

### CODE REVIEW

File: test/__tests/makeKey.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/*global test, module */
+module( 'makeKey()' );
+
+
+test( 'Returns keypath from a series of keys', t => {
+	var keys = ['a', 'b', 'c', 'd'],
+			expected = 'a.b.c.d';
```
Comment (narthur157) [inline test/__tests/makeKey.js]: Will fix this

File: src/Ractive/static/makeKey.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/**
+ * @param {Array} argument - series of keys
+ * @return {string} keypath
+ */
+export default function Ractive$makeKey() {
+	var keypath = "";
```
Comment (martypdx) [inline src/Ractive/static/makeKey.js]: Ractive is ES6ified, so you could do something like:

```js
Ractive$makeKey ( ...keys ) { return keys.join('.'); }
```

Comment (narthur157) [inline src/Ractive/static/makeKey.js]: Oh nice. This came from a non ES6 project, next commit does this now instead

**Comment by Rich-Harris** [comment]
Thanks for the PR @narthur157 - am curious as to what the use case is? Not doubting you, I'm just not seeing what the difference is between these two!

```js
Ractive.makeKey( a, b, c );
[ a, b, c ].join( '.' );
```

The second one is shorter (and to be rigorous, we'd have to call it `Ractive.makeKeypath()` :-) and doesn't require any API knowledge. Am I missing something?

**Comment by narthur157** [comment]
Use case for this would just be a convenience method over using join. Making keys is a fairly common thing. I made the function in my code partially because I was considering that I might need to sanitize values in my keypaths because some come from input, and just because the array syntax got a bit cluttery at times.

Though in my code I had makeKey globally defined, so having to prepend that with `Ractive` makes it a bit more cluttery.

**Comment by narthur157** [comment]
I'll close this, I agree with you

**Pull request closed without merging**